File: src/main/java/com/heliopolis/p3x972/spock/springboot/demo/entity/Thing.java
package com.heliopolis.p3x972.spock.springboot.demo.entity;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class Thing {
  private int id;
  private String name;
  private LocalDateTime updateDate;
}

File: src/main/kotlin/com/heliopolis/p3x972/spock/springboot/demo/entity/Thing.kt
package com.heliopolis.p3x972.spock.springboot.demo.entity

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonInclude.Include
import com.heliopolis.p3x972.spock.springboot.demo.domain.BaseThing
import java.time.LocalDateTime

@JsonInclude(Include.NON_EMPTY)
data class Thing(
    override var id: Int,
    override var name: String? = null,
    override var updateDate: LocalDateTime? = null
): BaseThing(id, name, updateDate)
